content/ko/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -0,0 +1,165 @@
+---
+title: 웹 UI (대시보드)
+content_template: templates/concept
+weight: 10
+card:
+  name: tasks
+  weight: 30
+  title: Use the Web UI Dashboard
+---
+
+{{% capture overview %}}
+대시보드는 웹 기반 쿠버네티스 유저 인터페이스이다. 대시보드를 통해 컨테이너화 된 애플리케이션을 쿠버네티스 클러스터에 배포할 수 있고, 컨테이너화 된 애플리케이션을 트러블슈팅 할 수 있으며, 클러스터 리소스들을 관리할 수 있다. 대시보드를 통해 클러스터에서 동작중인 애플리케이션의 정보를 볼 수 있고, 개별적인 쿠버네티스 리소스들을(예를 들면 디플로이먼트, 잡, 데몬셋 등) 생성하거나 수정할 수 있다. 예를 들면, 디플로이먼트를 스케일하거나, 롤링 업데이트를 초기화하거나, 파드를 재시작하거나 또는 배포 마법사를 이용해 새로운 애플리케이션을 배포할 수 있다.
+
+또한 대시보드는 클러스터 내 쿠버네티스 리소스들의 상태와 발생하는 모든 에러 정보를 제공한다.
+
+![Kubernetes Dashboard UI](/images/docs/ui-dashboard.png)
```

Comment:
```suggestion
![대시보드 UI](/images/docs/ui-dashboard.png)
```

Reply:
@sftim  Thanks! :)

Reply:
Sorry @sftim . I think it looks like a label that you suggested, so it doesn't have to translate Korean. It doesn't expose it at all. Watch deploy preview :)
So i won't apply this.

Reply:
This is the [alternate text](https://ko.wikipedia.org/wiki/Alt_%EC%86%8D%EC%84%B1), used for accessibility purposes. Ideally it would be localized.

Reply:
@sftim Yeah, I agree with that. But these images are not localized yet. And I also wait for these being localized. :)
Until then, I will leave it as it is.